lib/api/CAnomalyJob.cc

```diff
@@ -85,7 +85,9 @@ const std::string INTERIM_BUCKET_CORRECTOR_TAG("k");
 
 //! The minimum version required to read the state corresponding to a model snapshot.
 //! This should be updated every time there is a breaking change to the model state.
-const std::string MODEL_SNAPSHOT_MIN_VERSION("6.4.0");
+//! (The last breaking change was in 7.9 in lib/core/CPackedBitVector.cc in
+//! https://github.com/elastic/ml-cpp/pull/1340)
+const std::string MODEL_SNAPSHOT_MIN_VERSION("7.9.0");
```

Comment:
Can we add a comment that this is not breaking backwards compatibility but forwards compatibility as well? We probably should have a QA test: i.e. export model and try to load into the build with MODEL_SNAPSHOT_MIN_VERSION version. I suspect other changes have broken forwards compatibility before this one (although this will be the last).